### DISCUSSION TITLE
Unable to launch exe

### DISCUSSION THREAD

**Discussion by user36795** [streamlink/streamlink#5485] [Q&A]
### Description

I've been trying to update to the new 6.0.1 streamlink but after downloading streamlink-6.0.1-1-py311-x86_64.exe and running it nothing opens
I tried to uninstall and reinstall both Streamlink Twitch GUI and my old streamlink, i was able to redownload the twitch gui but the streamlink doesnt do anything after i allow it to run on windows
I made sure that i had the most updated python version but that still didnt fix it

**Comment by bastimeyer** [comment]
What you're describing is totally vague, without any actual useful information to help you.

First, this kind of issue/question doesn't really belong here, because Windows stuff gets built and distributed on the windows-builds repo, which is separate. Same with the Twitch GUI.

- What doesn't work? The Streamlink installer or Streamlink itself? What is the specific error message?
- Did you try to run Streamlink from the command-line shell?
- Do you have multiple versions installed separately? Run `where streamlink` from the command-line shell.
- Which Windows version are you using? Windows 7 users need to use the Python 3.8 builds, as described in the docs, readme and release notes.
- Is it a configuration issue in your Twitch GUI settings?

**Reply by user36795** [reply]
Ah sorry im super new to all of this and i thought this was meant to go here.
I believe it is the installer that does not work, after running the exe and clicking run anyway nothing opens
I uninstalled all of my previous streamlink versions
Windows 10, so i downloaded python 3.11

**Reply by bastimeyer** [reply]
> so i downloaded python 3.11

Python comes bundled with the installer and portable builds.
https://github.com/streamlink/windows-builds#streamlinkwindows-builds

> I believe it is the installer that does not work

Post the name of the executable which is not working.
